### PR TITLE
codemirror file view: Add search widget

### PR DESCRIPTION
**Pull request by fkling**
This adds CodeMirror's default search functionality. This works now
because the file view is focusable. When focused (e.g. by clicking into
it), meta+f will show CodeMirror's search input.

This currently uses it's own default style but we might be able to
either customize it via CSS or we can completely render our own UI (the
search extension allows to do that).

## Test plan

Enable CodeMirror file view, focus it, press meta+f to search.

## App preview:

- [Web](https://sg-web-fkling-file-view-search.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-prwnsaadbq.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Review by umpox — APPROVED** [review]
Very cool, +1 for customising the style though 😀

File: package.json [hunk 1]
```diff
@@ -368,12 +368,12 @@
     "@codemirror/lang-markdown": "^6.0.0",
     "@codemirror/language": "^6.2.0",
     "@codemirror/lint": "^6.0.0",
+    "@codemirror/search": "^6.0.1",
     "@codemirror/state": "^6.1.0",
     "@codemirror/view": "^6.0.2",
     "@lezer/highlight": "^1.0.0",
     "@mdi/js": "^6.7.96",
     "@microsoft/fetch-event-source": "^2.0.1",
-    "@opentelemetry/semantic-conventions": "^1.5.0",
```
Comment (umpox) [inline package.json:376]: Shouldn't be removed?

Comment (fkling) [reply package.json:376]: It looks like this is listed twice in the dependencies.

Comment (valerybugakov) [reply package.json:376]: Thanks, @fkling, I added them separately in two different PRs 😌

**Pull request merged**